apps/extension/src/background/keyring/types.ts

```diff
@@ -60,3 +60,8 @@ export type MnemonicValidationResponse = {
   isValid: boolean;
   error?: string;
 };
+
+export type SigningKey = {
+  pk?: string;
```

Comment:
it would be nice to rename to privateKey or secretKey, so it's not confused with publicKey :)